src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -168,9 +169,14 @@ def serialize(self, parent_map, root, warning_extra, params, snuba_event=None, e
                 event.event_id: event
                 for event in eventstore.get_events(
                     eventstore.Filter(
+                        start=params["start"],
+                        end=params["end"],
```

Comment:
Unrelated fix, this corrects an issue where we're doing a 90d query in snuba for these events.